seed-data-scripts/seed.js

```diff
@@ -0,0 +1,362 @@
+const axios = require('axios');
```

Comment:
same as the other script, it should stay in it's own directory.

Reply:
done